### PR TITLE
style: Update interface of badges in repository component

### PR DESCRIPTION
**Pull request by rkpasia**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | iPhone 6 (Emulator) |
| Bug fix?         | no      |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  |         |

---

## Screenshots

| Before   | After    |
| -------- | -------- |
|before|after|

## Description
Hello! In this short commit I changed the styles associated with the badges in the repository component. I wasn't happy with the previous interface, so I tried to play with it and the results can be seen in the screenshots :)

I haven't created an issue or discussed with anyone else about this before doing it. So if it isn't accepted I understand, it just seemed a good way to start contributing to the project :)

### CODE REVIEW

File: src/components/repository-profile.component.js [hunk 1]
```diff
@@ -98,18 +98,18 @@ const styles = StyleSheet.create({
...
     ...fonts.fontPrimary,
   },
-  badge: {
-    paddingTop: 3,
-    paddingBottom: 3,
-    marginTop: 5,
-    marginLeft: 27,
-    marginRight: 27,
-    borderWidth: 0.5,
+  badge_view: {
```
Comment (machour) [inline src/components/repository-profile.component.js]: badgeView would be me consistent with our code style

File: src/components/repository-profile.component.js [hunk 2]
```diff
@@ -98,18 +98,18 @@ const styles = StyleSheet.create({
...
+    padding: 2,
+    flexGrow: 0,
+    backgroundColor: colors.lighterBoldGreen,
+    marginLeft: 20,
+    marginRight: 20,
+    marginTop: 5,
+  },
+  badge: {
+    color: colors.white,
+    fontWeight: 'bold',
```
Comment (machour) [inline src/components/repository-profile.component.js]: Use ...fonts.fontPrimaryBold instead

**Comment by machour** [comment]
Hey, this is a fantastic way to start contributing, what a nice UI update :tada: :tada:

Left you some minor changes requests to stick to our coding habits. 

Could you also add yourself as a contributor using `yarn contributor:add` ? 

Thank you again for your PR!

**Comment by housseindjirdeh** [comment]
This is so great @rkpasia 😍 . I knew those tags could use a little bit of love, but couldn't place my finger on what exactly it needed to look a little better.

Aside from the tiny things @machour, this looks great. Also, please don't forget to add yourself as a contributor with `yarn contributors:add`!!

**Comment by rkpasia** [comment]
@machour thanks for the feedback! 😄 In the last commit I've updated the code for consistency.

@housseindjirdeh Thanks for the great words 😄I've added myself to the contributors list! Forgot about it ahah

I'll reach out in gitter about the possibility to further get involved in the project 😄

**Pull request merged**